doc/tutorials/use.md

```diff
@@ -159,7 +159,7 @@ lxc-start -n myContainerImage
 With LXD it's possible to run Windows VMs. All you need is a Windows ISO and a bunch of drivers.
 To make the installation a bit easier, `lxd-imagebuilder` added the `repack-windows` command. It takes a Windows ISO, and repacks it together with the necessary drivers.
 
-Currently, `lxd-imagebuilder` supports Windows 10, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022. The Windows version will automatically be detected, but in case this fails you can use the `--windows-version` flag to set it manually. It supports the values `w10`, `2k12`, `2k16`, `2k19` and `2k22` for Windows 10, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022 respectively.
+Currently, `lxd-imagebuilder` supports Windows 10, Windows 11, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022. The Windows version will automatically be detected, but in case this fails you can use the `--windows-version` flag to set it manually. It supports the values `w10`, `w11`, `2k12`, `2k16`, `2k19` and `2k22` for Windows 10, Windows 11, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022 respectively.
```

Comment:
Thanks for the improved docs. I went with the approach to have a table of supported versions, as it is a bit easier to read, maybe can serve as some inspiration: https://github.com/canonical/lxd-imagebuilder/pull/86/commits/d3c07039ccd344f15ec5b2717bff45541a77aab6

Reply:
Ah great, thanks for bringing support (via cherry-picks) for those EOL Windows versions. Indeed, your table's better and I'll simply rework the paragraph to mention it is autodetected but can be forced if needed.